components/notifier/src/quality_time_notifier.py

```diff
@@ -25,7 +25,7 @@ async def notify(log_level: int = None) -> NoReturn:
     )
     data_model = await retrieve_data_model(api_version)
     most_recent_measurement_seen = datetime.max.isoformat()
-    ready_to_send_metrics: List = []
+    outbox = Outbox([])
```

Comment:
My changes above make the empty list argument unnecessary.

```suggestion
    outbox = Outbox()
```